pages/instrumenting.md

```diff
@@ -109,27 +115,25 @@ Communicating trace information
 -------------------------------
 
 Trace information needs to be passed between upstream and downstream services in
-order to reassemble a complete trace. Five pieces of information are required:
+order to reassemble a complete trace.  Five pieces of information are required:
```

Comment:
supernit, but I think we should be consistent about single-spacing after periods, as opposed to double space typewriter convention